crates/nu-cmd-extra/src/extra/strings/format/command.rs

```diff
@@ -14,11 +14,11 @@ pub struct Format;
 
 impl Command for Format {
```

Comment:
```suggestion
impl Command for FormatPattern {
```
I'd prefer to be a little more thorough here and change the name. And then also change the name in the extra default context.

Reply:
Fixed.  Forgot to include the edited files in the previous force push.